src/components/v5/common/CompletedAction/partials/AddVerifiedMembers/AddVerifiedMembers.tsx

```diff
@@ -64,7 +64,10 @@ const AddVerifiedMembers = ({ action }: AddVerifiedMembersProps) => {
       <ActionDataGrid>
         <ActionTypeRow actionType={action.type} />
         <AddRemoveRow actionType={action.type} />
-        <DecisionMethodRow isMotion={action.isMotion || false} />
+        <DecisionMethodRow
+          isMotion={action.isMotion || false}
+          isMultisig={action.isMultiSig || false}
+        />
```

Comment:
I'd say this is repeated quite a lot, and maybe we could create something to reuse.

But then again, it's a small component and I guess, it's just the nature of how the sidebar forms are structured